server/events.go

```diff
@@ -1063,14 +1063,14 @@ func (s *Server) remoteLatencyUpdate(sub *subscription, _ *client, subject, _ st
 
 	// So we have not processed the response tracking measurement yet.
 	if m1 == nil {
-		acc.mu.Lock()
+		si.acc.mu.Lock()
```

Comment:
Seeing that, wonder if the s.sendInternalAccountMsg() below is for `acc` or `si.acc`?

Reply:
I believe the service import's account protects the change, the export account (a) is where we send the metrics.